crates/shielded_token/src/conversion.rs

```diff
@@ -298,36 +323,8 @@ where
     // reward tokens with the zeroth epoch to minimize the number of convert
     // notes clients have to use. This trick works under the assumption that
     // reward tokens will then be reinflated back to the current epoch.
```

Comment:
```
    // Construct MASP asset type for rewards. Always deflate and timestamp
    // reward tokens with the zeroth epoch to minimize the number of convert
    // notes clients have to use. This trick works under the assumption that
    // reward tokens will then be reinflated back to the current epoch.
```
this comment should be moved inside the def of `encode_reward_asset_types`

Reply:
Done, thanks!